external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -115,66 +120,588 @@ To reuse or extend existing dimensions, attributes, units, or measures, provide
+
+```json
+{ ...
+   "columns": {
+      "Number of Arthur's Bakes": {
+         "type": "observations",
+         "data_type": "integer",
+         "unit": {
+            "label": "Count"
+         },
+         "measure": {
+            "label": "Number of Stores"
+         }
+      },
+      "Revenue": {
+         "type": "observations",
+         "data_type": "decimal",
+         "measure": {
+            "label": "Revenue"
+         }
+      },
+      "Revenue Units": {
+         "type": "units",
+         "describes_observations": "Revenue"
+      }
+   }
+}
+```
+
+## Measures and Units (for the guide)
+
+Measures can either be attached to a Measure Column if there are multiple different measures in your data set, or to an Observation column's `measure` field if all observations in the cube have the same measure.
+Units can either be attached to a Unit Column if there are a mixture of units in your data set, or to an Observation column if all observations in the cube have the same unit.
+
+The definition of measures and units columns is similar in that it depends on the number of measures/units present in the data set. If only one measure/unit is present, then the measure/unit can be defined by being attached to the Observation column, by entering them into their own field in the Observation column's definition. When multiple measures/units are present in the data set, they can be defined in their own column so that the column may clearly show which of the possible measures is being counted, or which unit is being used to measure each value in observations.
+
+## Measure and Unit Columns Configuration (for the guide)
+
+Measure and unit columns are treated slightly differently to dimension, attribute, and observation columns. Measure and unit columns contain references to discrete units and measures. In both cases by defining `"type": "measures"` or `"type": "units"` provides the same behaviour. Do not put measures in unit columns or units in measure columns.
+
+Measure and unit columns share some fields in that they can be populated directly in the column's definition with the `values` field, allowing a dictionary to be input that defines the details of the unit/measure. Existing column definitions can also be reused to quickly define measures and units, or serve as a "base" for a new column to be created on.
+
+TODO: Find out if cell_uri_template in measures/units has any distinctions compared to when it is used as a field in other column definitions, to avoid overlap/writing stuff about this field that is explained elsewhere in the guide.
```

Comment:
`cell_uri_template` has the same meaning when used in a units or measures column as elsewhere. It should only be used where the user wants to 100% reuse existing measure or units that are defined elsewhere.